pkg/clientinteractor/interactor.go

```diff
@@ -592,20 +540,6 @@ func (pi *PSQLInteractor) DropShardingRule(ctx context.Context, id string) error
 	return pi.CompleteMsg(0)
 }
 
-// TODO : unit tests
-func (pi *PSQLInteractor) AddShardingRule(ctx context.Context, rule *shrule.ShardingRule) error {
```

Comment:
Lets drop this: https://github.com/pg-sharding/spqr/pull/517/files#diff-6c38386446aa700a6b936e66206e0d5306617e4958380aa53d0ffaa3e53e9eceR530